hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/snapshot/SnapshotDiffManager.java

```diff
@@ -565,6 +564,8 @@ private synchronized SnapshotDiffResponse submitSnapDiffJob(
           volumeName, bucketName, fromSnapshotName, toSnapshotName,
           forceFullDiff));
       updateJobStatus(jobKey, QUEUED, IN_PROGRESS);
+      // DO NOT decrement snapshot instance ref count, executor is responsible
```

Comment:
This comment confuses me.  Which snapshot instance is being referred to here?

Reply:
Ah. This is a left-over comment that belongs to the old approach before try-with-resources. Removed.